rxbinding-leanback-v17/src/main/java/com/jakewharton/rxbinding2/support/v17/leanback/widget/SearchBarSearchQueryChangeEventsOnSubscribe.java

```diff
@@ -0,0 +1,59 @@
+package com.jakewharton.rxbinding2.support.v17.leanback.widget;
+
+import android.support.v17.leanback.widget.SearchBar;
+import com.jakewharton.rxbinding2.support.v17.leanback.widget.SearchBarSearchQueryEvent.Kind;
+import io.reactivex.Observable;
+import io.reactivex.Observer;
+import io.reactivex.android.MainThreadDisposable;
+
+import static io.reactivex.android.MainThreadDisposable.verifyMainThread;
+
+final class SearchBarSearchQueryChangeEventsOnSubscribe
+    extends Observable<SearchBarSearchQueryEvent> {
+  final SearchBar view;
```

Comment:
In RxJava it's standard to not make them private I don't know what Jake wants though so I'll leave at is